rfcs/20200205-standalone-keras-repository.md

```diff
@@ -0,0 +1,351 @@
+cache might help speed up the follow-up builds, the initial time cost is too 
+high for regular software development workflows. Internally, Google has a
+distributed build and caching service, which Googlers heavily rely on,
+that can build TensorFlow and run all Keras tests within 5 mins. Sadly,
+we can't expose this to external contributors.
+
+Currently, any contribution to Keras code will require building all of
+TensorFlow, which is quite expensive to do for average users.
+Having a separate repository will allow the Keras package to be built
+without building TensorFlow. This should greatly improve the 
+velocity of open-source developers when they contribute to Keras code.
+
+### Community Benefit
+
+The difficulty of building TensorFlow from scratch in order to make a PR
+to Keras code has been a significant source of issues:
+
+* It discouraged contributions, since many external developers couldn't test
+their changes and make sure they were correct.
+* External developers would send unverified PRs, and Google reviewers spend time back 
+and forth, fixing the PR. Sometimes PR is just not moving forward because of the
+lengthy feedback loop.
+
+With the new standalone Keras repository, external contributors
+should experience much shorter turn-around time when 
+building/testing Keras, since they don't need to build TensorFlow anymore.
+This should  have a positive impact on building a vibrant open-source
+developer community.
+
+In addition, by getting the Keras team at Google to start developing Keras
+using the same public tools and infrastructure as third-party developers,
+we make the development process more transparent and more community-oriented.
+
+### TensorFlow API modularity
+
+There are other side-benefits if we split the repository. Currently, Keras
+has to rely on a number of private TensorFlow APIs. However, a litmus test
+of the quality of the public TensorFlow low-level APIs is that they should
+be strictly sufficient to a higher-level API like Keras.
+After splitting the repository, Keras will have to import TensorFlow and 
+rely exclusively on public APIs. If Keras still ends up using TensorFlow
+private features, it  might be an indication of tight coupling of
+implementation details. If certain private features are extensively used,
+we might want to consider exposing them  as public low level API.
+
+This design is also aligned with thee design for
+[Modular TensorFlow](https://github.com/tensorflow/community/blob/master/rfcs/20190305-modular-tensorflow.md), 
+which splits the TensorFlow project into smaller components that are not
+tightly coupled together.
+
+
+## Design Proposal
+
+### New location of the code
+
+GitHub: the code will live at [keras-team/keras](https://github.com/keras-team/keras), 
+joining the other Keras SIG projects and replacing the current external Keras 
+codebase. `tf.Keras` will also replace Keras on PyPI.
+
+Also considered: `tensorflow/keras`.
```

Comment:
Updated for the section for repo comparison. I think the we lean towards keras-team/keras since it already has a lot of uses/traffic/stars and incoming references.

Reply:
Will this be the only "core project" outside Tensorflow/? Have we other sigs with out of Tensorflow/ repo?
Cause It will be really hard to explain why this will be the only "brand" that It is so core but still something like an external project.
I think the story telling will not work so much.